BedrockCore.cpp

```diff
@@ -75,6 +75,22 @@ void BedrockCore::prePeekCommand(unique_ptr<BedrockCommand>& command) {
     try {
         try {
             SDEBUG("prePeeking at '" << request.methodLine << "' with priority: " << command->priority);
+
+            // Regularize case
+            list<string> lowerCaseAttributes = {"email", "partnerUserID", "partnerName"};
```

Comment:
Why are we doing this in Bedrock and not in the Auth plugin? For separation of concerns, I don't think Bedrock(Core) should have knowledge about its plugins (especially ones in private repos, as we are in a public repo here)